content/lessons/table_of_contents.json

```diff
@@ -1,36 +1,53 @@
 {
   "id": "table-of-contents",
-  "coolness": "very-cool",
-  "chapters": {
-    "one": {
-      "subchapter_one_one": [
+  "chapters": [
+    {
+      "title": "The first chapter",
+      "subchapters": [
         {
-          "post": "./lesson_one.md"
+          "title": "The rainbow",
+          "entries": [
+            {
+              "entry": "./lesson_one.md"
+            },
+            {
+              "entry": "./lesson_two.md"
+            },
+            {
+              "entry": "./lesson_three.md"
+            }
+          ]
         },
         {
-          "post": "./lesson_two.md"
-        },
-        {
-          "post": "./lesson_three.md"
+          "title": "The mountain",
+          "entries": [
+            {
+              "entry": "./lesson_four.md"
+            }
+          ]
         }
-      ],
-      "subchapter_one_two": [
+      ]
+    },
+    {
+      "title": "The second chapter",
+      "subchapters": [
         {
-          "post": "./lesson_four.md"
```

Comment:
This is not a blog, but a documentation. I think `"post"` does not make sense. I propose "`entry`". Happy to discuss naming here.